packages/api/prisma/migrations/20220713133350_init/migration.sql

```diff
@@ -0,0 +1,745 @@
+
+-- AddForeignKey
+ALTER TABLE "images.focal-point" ADD CONSTRAINT "images.focal-point_imageId_fkey" FOREIGN KEY ("imageId") REFERENCES "images"("id") ON DELETE CASCADE ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "comments.revisions" ADD CONSTRAINT "comments.revisions_commentId_fkey" FOREIGN KEY ("commentId") REFERENCES "comments"("id") ON DELETE SET NULL ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "comments" ADD CONSTRAINT "comments_userID_fkey" FOREIGN KEY ("userID") REFERENCES "users"("id") ON DELETE SET NULL ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "invoices.items" ADD CONSTRAINT "invoices.items_invoiceId_fkey" FOREIGN KEY ("invoiceId") REFERENCES "invoices"("id") ON DELETE CASCADE ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "invoices" ADD CONSTRAINT "invoices_subscriptionID_fkey" FOREIGN KEY ("subscriptionID") REFERENCES "subscriptions"("id") ON DELETE CASCADE ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "invoices" ADD CONSTRAINT "invoices_userID_fkey" FOREIGN KEY ("userID") REFERENCES "users"("id") ON DELETE SET NULL ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "member.plans.payment-methods" ADD CONSTRAINT "member.plans.payment-methods_memberPlanId_fkey" FOREIGN KEY ("memberPlanId") REFERENCES "member.plans"("id") ON DELETE CASCADE ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "member.plans" ADD CONSTRAINT "member.plans_imageID_fkey" FOREIGN KEY ("imageID") REFERENCES "images"("id") ON DELETE SET NULL ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "navigations.links" ADD CONSTRAINT "navigations.links_navigationId_fkey" FOREIGN KEY ("navigationId") REFERENCES "navigations"("id") ON DELETE CASCADE ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "pages" ADD CONSTRAINT "pages_publishedId_fkey" FOREIGN KEY ("publishedId") REFERENCES "pages.revision"("id") ON DELETE SET NULL ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "pages" ADD CONSTRAINT "pages_pendingId_fkey" FOREIGN KEY ("pendingId") REFERENCES "pages.revision"("id") ON DELETE SET NULL ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "pages" ADD CONSTRAINT "pages_draftId_fkey" FOREIGN KEY ("draftId") REFERENCES "pages.revision"("id") ON DELETE SET NULL ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "payments" ADD CONSTRAINT "payments_paymentMethodID_fkey" FOREIGN KEY ("paymentMethodID") REFERENCES "payment.methods"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "peerProfiles" ADD CONSTRAINT "peerProfiles_logoID_fkey" FOREIGN KEY ("logoID") REFERENCES "images"("id") ON DELETE SET NULL ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "sessions" ADD CONSTRAINT "sessions_userID_fkey" FOREIGN KEY ("userID") REFERENCES "users"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "subscriptions.periods" ADD CONSTRAINT "subscriptions.periods_invoiceID_fkey" FOREIGN KEY ("invoiceID") REFERENCES "invoices"("id") ON DELETE CASCADE ON UPDATE CASCADE;
```

Comment:
Shouldn't we `set null` here? and allow null for period? or its always connected.